{{ cookiecutter.__dirname }}/.github/workflows/build.yml

```diff
@@ -54,6 +54,17 @@ jobs:
         pre-commit run --all-files
     - name: Build assets
       run: circuitpython-build-bundles --filename_prefix {% raw %}${{ steps.repo-name.outputs.repo-name }}{% endraw -%} {% if cookiecutter.target_bundle == 'Community' %} --package_folder_prefix {% if cookiecutter.library_prefix -%} {{ cookiecutter.library_prefix | lower | replace(' ', '_') }}_ {%- else %}""{%- endif -%}{%- endif %} --library_location .
+    - name: Check Sizes
+      run: |
+        git clone https://github.com/FoamyGuy/CircuitPython_Memory_Tools.git
```

Comment:
Should that move to https://github.com/circuitpython ?

Reply:
I do like that idea. I'll cookiecut a new one to get the relavent boilerplate stuff and move this code into it, then put it in the circuitpython org. Probably sometime tomorrow I can work on that.

Reply:
Repo now lives here: https://github.com/circuitpython/CircuitPython_Org_size_tools

Reply:
Still need to update this PR to point to the new one. Will make a new commit with that once I've got the conditions mentioned in the other comment worked out.